### ISSUE TITLE
Misplaced Lab in Curriculum? 

### ISSUE THREAD

**Issue by jdbean** [learn-co-curriculum/collections_practice_vol_2#20]
In Web Dev Fundamentals this lab is placed at the "Enumerables" section of the course. However, the lab clearly is testing skills relating to 

> iteration, arrays, hashes, nested data structures

hashes are covered in the subsequent chapter and  nested data structures may not be covered at all in this particular course. Thus, I think this lesson is likely misplaced as students are not equipped to complete all the tasks required by the lab at the stage at which it is presented to them.

**Comment by jdbean** [comment]
Apparently this is a duplicate of #1 #2 #3 #4 #5 #6 #8 #9 #10 #12 #13 #14 #15 #16 #17 #18 #19 

Very disappointing to see that this bug has resurfaced (presuming it was addressed in the first place) and has been open for so long.

**Comment by jdbean** [comment]
@aviflombaum @sgharms Looks like maybe the issue queue for this repo has fallen by the wayside?

**Comment by aviflombaum** [comment]
Thanks for the bump.

**Comment by sgharms** [comment]
Good catch, @jdbean.  Thank you for spotting this issue and providing feedback.

We apologize for any frustration you might have encountered during this process
but thank you for helping us ensure that those who follow in your path will not
encounter the same problems that you faced.

If you do not believe that this issue has been addressed, please re-open this
issue. 💙

Fixes #19 
Fixes #18 
Fixes #16 
Fixes #15 
Fixes #13 
Fixes #12

**Comment by sgharms** [comment]
Let me check on your status specifically. Thanks for the news.

**Comment by sgharms** [comment]
@jdbean Just to close the loop per your message from the other night, I think you should find it disappeared now. Be sure to refresh your browser page.

Steven

**Comment by sgharms** [comment]
OK @jdbean, I think I've got it recursively deleted in all the places. I
_think_ we're good now. Can you confirm?

Thanks for all the help working through this issue.

Steven

**Comment by jdbean** [comment]
@sgharms Of course!

I can happily confirm that the lesson no longer appears out of place in the _Enumerables_ section of my curriculum.

Per your above message, the lesson also no longer appears in the _More on Hashes_ section.

**Comment by sgharms** [comment]
@jdbean: I'd love your perspective here. Is the challenge OK to keep in the
**More on Hashes** section? I _think_ it's reasonable to expect that those
traversals be possible, but I'd appreciate your insight here.

Or is there some "scaffolding" we could provide in the README to make it
friendlier? I like its variety but I also want to make sure we're keeping
learners stretching _just enough_.

Steven

**Comment by jdbean** [comment]
I think the challenge is totally fine to keep in. There were certainly more or equally challenging traversal labs earlier on the curriculum.

I do, however, think the fleshing out the README or at least the commented prompts would be helpful. I found some of the more advanced function challenges to be a little vague in terms of what I had to do to solve them and didn't find the descriptive text in rspec to be quite enough at times.
